src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -345,4 +353,52 @@ public function theFirstOrderShouldHaveNumber(string $number)
 
         Assert::same($firstItem['number'], str_replace('#', '', $number));
     }
+
+    /**
+     * @When /^I want to modify a customer's billing address of this order$/
+     */
+    public function iWantToModifyACustomerBillingAddress(): void
+    {
+        $this->client->buildUpdateRequest(
+            Resources::ADDRESSES,
+            (string) $this->sharedStorage->get('order')->getBillingAddress()->getId(),
+        );
+    }
+
+    /**
+     * @When /^I specify their (?:|new )billing (address as "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)" for "([^"]+)")$/
+     */
+    public function iSpecifyTheirBillingAddressAsFor(AddressInterface $address): void
```

Comment:
If it's not established scheme, then `As` sounds to me better than `AsFor`.
```suggestion
    public function iSpecifyTheirBillingAddressAs(AddressInterface $address): void
```

Reply:
The general rule is to include the "uncaptured parts" of the regular expression in the method name, but tweaking it to sound more natural shouldn't be a problem